packages/vira/src/elements/vira-dropdown.element.ts

```diff
@@ -0,0 +1,561 @@
+import {PartialAndUndefined, joinWithFinalConjunction} from '@augment-vir/common';
```

Comment:
this file is 500 line long, consider moving these utils functions out of this file.